enterprise/app/trends/trends.tsx

```diff
@@ -260,6 +264,8 @@ export default class TrendsComponent extends React.Component<Props> {
                 formatHoverLabel={this.formatLongDate}
                 formatHoverValue={(value) => (value || 0) + " users"}
                 name="users with builds"
+                clickableBars={true}
```

Comment:
Rather than passing in a boolean and a hash and having the chart component know how to navigate - I think it would be cleaner to pass in an `onDateClicked` function. 

If this function prop is set, the bars become clickable - and then the chart doesn't have to know what a router is or what a hash is.

Reply:
Is there a way to do this without passing in the hash? I'm not completely familiar with the intricacies but if i pass an onValueClicked function into a chart, is there a way to pass a hash parameter in with the function and then put in the date parameter later?

Reply:
Got it!